### PR TITLE
Add release notes for the new IA features (FAB, Me and Reader)

### PR DESCRIPTION
**Pull request by Gio2018**
REF: [IA Project](https://github.com/wordpress-mobile/WordPress-iOS/projects/15)

To test:
- Check that the release notes are updated with content related to FAB, Me and Reader

PR submission checklist:

- [x] ~~I have considered adding unit tests where possible.~~ NA
- [x] ~~I have considered adding accessibility improvements for my changes.~~ NA
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,5 +1,9 @@
 14.9
 -----
+* Streamlined navigation: now there are fewer and better organized tabsposting shortcuts and more, so you can find what you need fast.
```
Comment (frosty) [inline RELEASE-NOTES.txt:3]:
```suggestion
* Streamlined navigation: now there are fewer and better organized tabs, posting shortcuts and more, so you can find what you need fast.
```

File: RELEASE-NOTES.txt [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+* My Site: the "Add Posts and Pages" features has been moved: there is a new "Floating Action Button" in "My Site" that lets you create a new post or page without having to navigate to another screen.
```
Comment (frosty) [inline RELEASE-NOTES.txt:4]:
```suggestion
* My Site: the "Add Posts and Pages" features has been moved. There is a new "Floating Action Button" in "My Site" that lets you create a new post or page without having to navigate to another screen.
```

File: RELEASE-NOTES.txt [hunk 3]
```diff
@@ -1,5 +1,9 @@
...
+* My Site: the "Me" section has been moved: there is a new button on the top right of "My Site" that lets you access the "Me" section from there.
```
Comment (frosty) [inline RELEASE-NOTES.txt:5]:
```suggestion
* My Site: the "Me" section has been moved. There is a new button on the top right of "My Site" that lets you access the "Me" section from there.
```

**Comment by Gio2018** [comment]
> Just a few small tweaks!

Done!! Thanks for reviewing!!

**Pull request merged**